frontend/front/src/pages/Public/locales/en.json

```diff
@@ -0,0 +1,237 @@
+        },
+        "item2": {
+          "title": "About Us",
+          "paragraph": "The Boston Heat Pump Accelerator is an initiative of the Urban League of Eastern Massachusetts, in partnership with local and national organizations.",
+          "button": "Learn more"
+        }
+      },
+      "testimonials": {
+        "heading1": "Testimonials",
+        "text1": "“Wow, it blew my mind. Normally the bill is anywhere from $400 to $500, and it was more like $200. I was like, let me look at that again!”",
+        "text2": "Stories from other Massachusetts residents about their heat pumps.",
+        "button": "read more",
+        "text3": "Resident"
+      },
+      "cardBenefitsSection": {
+        "heading1": "Benefits of Heat Pumps",
+        "subtitle1": "Learn more about the",
+        "link": "Benefits of Heat Pumps",
+        "item1": {
+          "title": "Lower energy bills",
+          "text1": "Heat pumps are so energy efficient that they can lead to significant savings on monthly utility bills and lower maintenance, repair and replacement costs."
+        },
+        "item2": {
+          "title": "Comfort at home",
+          "text1": "The same heat pump that helps to cool in the summer can then provide warmth in the winter- it's one system that is efficient, quiet, and convenient, providing comfort throughout the home."
+        },
+        "item3": {
+          "title": "Stronger communities",
+          "text1": "By helping to lower utility bills, heat pumps can lower the cost of living for residents, which in turn can support residents living on fixed- or low-incomes to remain in their homes longer, strengthening communities."
+        },
+        "item4": {
+          "title": "Cleaner air",
+          "text1": "Heat pumps use electricity that can be produced by renewable sources. As the grid becomes more green, heat pumps become much more eco-friendly than gas or oil-fueled systems. With a single “green-powered” unit that heats and cools, your home’s carbon footprint is as low as it can go."
+        }
+      },
+      "partners": {
+        "heading1": "Our Partners",
+        "text1": "The Boston Heat Pump Accelerator is an initiative of the Urban League of Eastern Massachusetts, in partnership with local and national organizations.",
+        "link": "Learn more"
+      }
+    },
+    "global-labels": {
+      "survey": "survey",
+      "learn-more": {
+        "value": "Learn More",
+        "items": {
+          "benefits-heat-pumps": "Benefits of Heat Pumps",
+          "about-us": "About Heat Pumps",
+          "testimonials": "Testimonials",
+          "about-bhpa": "About BHPA"
+        }
+      },
+      "get-involved": "Get Involved",
+      "button-get-heat-pump": "GET A HEAT PUMP",
+      "locales": {
+        "english": "English (United States)",
+        "portuguese": "Portuguese (Brazil)",
+        "spanish": "Spanish (Latin America)",
+        "creole": "Creole (Haiti)"
```

Comment:
I would have it like this in all four language files so the language user can find their language:
"english": "English (United States)",
"portuguese": Português (Brasil)",
"spanish": "Español (Latinoamérica)",
"creole": "Kreyòl (Ayiti)"